### ISSUE TITLE
*Bean Machine Local Deliveries or Increased Income*

### ISSUE THREAD

**Issue by Brookiedoughh** [InZidiuZ/op-framework-issue-tracker#2188]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes

**Summary:**
Write a summary of what your suggestion is about.
One of the main issues roleplaying as the Bean Machine owner is the lack of people that want to commit to role playing as a barista, mainly due to the lack of income. I believe adding local deliveries to Bean Machine will add traffic going through Bean Machine increasing the roleplay and provide a stable income for the employees. This will also lower the amount of people quitting and having a stable roster. If the lack of roleplay is a concern, I would be willing to come up with a system that involves having so many hours in the shop in order to complete local deliveries. 

**Reason:**
Why would your suggestion be a good change to the framework?
The delivery add would solve the problem of employee retention due to lack of income when rp is not flowing hot. Deliveries would be giving people the option to have something when rp is not there.  If employees are not making enough, they will just leave to grind something else since you are locked into the job. I can say for certain this will increase the foot traffic in Bean Machine and provide more roleplay in the city.

**Comment by RetroCharles** [comment]
I've heard about the income for BM employees being a bit too low compared to other source of incomes. 
Employees have to use their paycheck to purchase in-store items and upcharge for little profit. With that alone, bean machine struggles to keep a consistent roster. Having a secondary source of income such as local deliveries will increase to the longevity of BM employees and also provide more consistent flow of traffic/activity at the shop. Having to constantly lose employees who were just hired less than a couple of weeks or days can drain and demotivate the owner and managers
for something that is not in their control. 

There are more great topics and points made in this discussion that was posted in November #2063

**Comment by Brookiedoughh** [comment]
If local deliveries are not an option to be implemented, I would like to propose a possible pay increase for the employees of Bean Machine. Burgershot making an average $32 hourly, each item they sell, plus their local deliveries, give their employees a stable income for basic necessities and then some.
After brainstorming with a few Bean Machine staff in the city, this is what my starting proposal is. The starting pay for a barista would be $40, the bartender position $45, management $55, and owner $70. This would be a $20 increase for all positions for Bean Machine employees. In addition, having the prices of our "stock" lowered would also help with the amount of income gained. This idea would be if local deliveries is absolutely not an option currently. If later down the road it could be added, I would understand the need for a restructure of Bean Machine pay. It is hard to sit in the shop for hours with low hourly pay and have days when customers are not in sight. Having a higher pay would help fill in the financial gap with no having local deliveries.

**Comment by lildiavolo** [comment]
Bean machine does 100% need some love. It's honestly sad to say this, but I do forget that place exists most of the time until I pass by it or have to use it for directions. A big reason for this is I simply am not used to it being a place where active RP takes place. It's a lot more active recently and I've stopped by a couple times and I've enjoyed it. I do however fear that what you're saying is right and without any changes, the place will just die out again regardless of anyones best efforts. 
If burgershot can have local deliveries, so does bean machine. way more people will be interested in having a position there if that's the case. 
+1
